terraform/executor/Dockerfile-0.13

```diff
@@ -5,14 +5,14 @@ RUN cd tfmask && make && make go/build
 
 FROM debian:buster-slim
 
-ARG TF_VERSIONS="0.13.0 0.13.1 0.13.2"
+ARG TF_VERSIONS="0.13.0 0.13.1 0.13.2 0.13.3"
 ARG DEFAULT_TF_VERSION=0.13.2
 ARG GCLOUD_VERSION=279.0.0
 ARG AWSCLI_VERSION=1.18.134
 ARG HELM2_VERSION=2.16.10
 ARG HELM3_VERSION=3.3.1
 ARG TFSWITCH_VERSION=0.8.832
-ARG TF_AIVEN_VERSIONS="1.0.19 1.0.20 1.1.0 1.1.1 1.1.2 1.1.3 1.1.4"
+ARG TF_AIVEN_VERSIONS="2.0.6"
```

Comment:
this will force users of `0.13` to upgrade to `v2`

as per #161, could this be

```
ARG TF_AIVEN_VERSIONS="1.0.19 1.0.20 1.1.0 1.1.1 1.1.2 1.1.3 1.1.4 1.3.5"
```

or 

```
ARG TF_AIVEN_VERSIONS="1.3.5"
```

i.e. because is using 0.13 they can download v2 via terraform 0.13 support

Reply:
I don't expect anyone to be using this provider tbh and we should be using this version rather than the old one.

Reply:
I can't see anyone in ovo using this image currently, so it's a moot point

https://github.com/search?q=org%3Aovotech+terraform-0_13&type=code

i'll resolve this convo

Reply:
seen the latest change, looks like the best way to go imo